atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -560,6 +561,13 @@ def get_build(self, metadata):
                 else:
                     extra['filesystem_koji_task_id'] = task_id
 
+        help_result = self.workflow.prebuild_results.get(AddHelpPlugin.key)
+        if help_result is not None:
+            try:
+                extra['image']['help'] = self.workflow.prebuild_plugins_conf[AddHelpPlugin.key]['help_file']
+            except:
+                extra['image']['help'] = DEFAULT_HELP_FILENAME
```

Comment:
Good question. Possible ways of handling this:

- empty string, so an empty string means there is no help to find
- use `None`, so that a null value means there is no help to find
- omit the key altogether, so the absence of the key means there is no help to find